src/main/cpp/subsystems/DooHickey.cpp

```diff
@@ -120,6 +125,68 @@ void DooHickey::Periodic() {
       colorString = "Unknown";
     } 
 
+    /*Tracks the number of times a specific color is detected (in this case blue), so that we can 
+    * know when rotation control is over. We should see each color 2 times a rotation and the goal is 3
+    * rotations.
+    */ 
+    if (colorCounter >= 6) {
+      IsRotationControlFinished = true;
+    }
+
+    //Code for stopping the motor after Rotation Control is finished AND the specified Color is reached.
+    gameData = frc::DriverStation::GetInstance().GetGameSpecificMessage();
```

Comment:
I think this is going to be moved. I think the message only comes through when it's time to spin the wheel.